Jenkinsfile

```diff
@@ -28,9 +28,37 @@ pipeline {
             }
         }
 
-        stage('Deploy everything') {
+        stage('Setup network') {
             steps {
-                sh "./run.sh"
+                sh "./run.sh deploy_network"
+            }
+        }
+        stage('Setup hosts') {
+            parallel {
+                stage('Setup CaaSP') {
+                    steps {
+                        sh "./run.sh deploy_caasp"
+                        sh "./run.sh enroll_caasp_workers"
+                        sh "./run.sh setup_caasp_workers_for_openstack"
```

Comment:
Actually I think `setup_caasp_workers_for_openstack` cannot run before `deploy_ccp_deployer`, see `osh-deployer-in-hosts.yml` task

Reply:
Fixed